### DISCUSSION TITLE
[BUG]: Create or update the `source` field in the MySQL table does not take effect

### DISCUSSION THREAD

**Discussion by blust** [phalcon/cphalcon#15614] [Q&A]
```php
FooModel::setSource("bar");
FooModel::assign([
        "x"=>"xx",
        "source"=>1
    ]);
FooModel::create()
```
create or update the `source` field does not change，Please talk about how to resolve keyword conflicts

Phalcon version 4.0.0
PHP Version 7.4

**Comment by Jeckerson** [comment]
Try to check if `create()` returned true, if no, use code below as example:

```php
$model = new FooModel();
$model->assign(['source' => 1]);
if (!$model->create) {
    foreach ($model->getMessages() as $error) {
        echo (string)$error;
    }
}
```

**Reply by blust** [reply]
Use create() to return true, but the source field in the MySQL table is not the specified value, and other fields in the update() record return false. Just change the `source` field name to another name, such as ` source_foo`

**Reply by Jeckerson** [reply]
Try with latest Phalcon v4.1.2 and see if issue still occur.

**Reply by blust** [reply]
I tried to verify with V4.1.2 and found the same problem

**Reply by Jeckerson** [reply]
@blust Could you please create issue with `bug` template and fill all necessary information with specific code example + DB schema?
